services/frontend/src/components/AnalysisVisualization.vue

```diff
@@ -1,20 +1,136 @@
 <template>
-  <div>
-    <!-- create graphs here -->
-    <!-- this is a mock -->
-    <img src="mock_files/mock_analysis_graph.png"/>
+  <div v-if="analysisStared" class="sector flex-column-items-centered card"> Rozpocznij analizę </div>
+
+  <div v-else class="sector flex-column-items-centered card">
+
+    <div class="flex-row-items-centered" style="margin-block-end: 30px">
+
+      <table style="margin-inline-end: 30px">
+        <tr>
+          <td>Pesel </td>
+          <td>01234567890 </td>
+        </tr>
+        <tr>
+      <td>Imie</td>
+      <td>Daniel</td>
```

Comment:
If you have time you can already change it to display a variable value. It will save time l8r when we will be connecting it with backend

Reply:
i think this should be done in different PR, but i have time and can do it now